web/thesauruses/nim/classes.json

```diff
@@ -0,0 +1,127 @@
+{
+  "meta": {
+    "language": "nim",
+    "language_version": "1.4",
+    "language_name": "Nim"
+  },
+  "categories": {
+    "Defining Classes": [
+      "normal_class",
+      "abstract_class",
+      "interface",
+      "read_only_class",
+      "static_class",
+      "inner_class"
+    ],
+    "Adding Private and Public Members": [
+      "private_variables",
+      "protected_variables",
+      "public_variables",
+      "static_variables",
+      "private_functions",
+      "protected_functions",
+      "public_functions",
+      "static_functions"
+    ],
+    "Extending and Implementing Classes": [
+      "extends_class",
+      "extending_interface",
+      "calling_superclass_functions"
+    ],
+    "Creating Objects and Polymorphism": [
+      "instantiating_object",
+      "instantiating_polymorphic_object"
+    ],
+    "Constructors and Deconstructor": [
+      "implement_constructor"
+    ]
+  },
+  "classes": {
+    "normal_class": {
+      "name": "Normal class",
+      "code": "type Name = object"
+    },
+    "abstract_class": {
+      "name": "Abstract class",
+      "code": "type Name = concept"
+    },
+    "interface": {
+      "name": "Interface",
+      "code": "type Name = concept"
+    },
+    "read_only_class": {
+      "name": "Read-only class",
+      "code": "type Name = object"
+    },
+    "static_class": {
+      "name": "Static class",
+      "code": "Types are static"
```

Comment:
Should this be `"comment": "Types are static"` instead?

Reply:
Fixed.